tests/parm/model_configure_hafsv2.IN

```diff
@@ -0,0 +1,52 @@
+print_esmf:              .false.
+start_year:              @[SYEAR]
+start_month:             @[SMONTH]
+start_day:               @[SDAY]
+start_hour:              @[SHOUR]
+start_minute:            0
+start_second:            0
+nhours_fcst:             @[FHMAX]
+RUN_CONTINUE:            .false.
+ENS_SPS:                 .false.
+dt_atmos:                @[DT_ATMOS]
+calendar:                'julian'
+memuse_verbose:          .false.
+atmos_nthreads:          @[THRD]
+use_hyper_thread:        .false.
+ncores_per_node:         @[TPN]
+debug_affinity:          .true.
+restart_interval:        @[RESTART_INTERVAL]
+output_1st_tstep_rst:    .false.
+
+quilting:                @[QUILTING]
+write_groups:            @[WRITE_GROUP]
+write_tasks_per_group:   @[WRTTASK_PER_GROUP]
+num_files:               @[NUM_FILES]
+filename_base:           @[FILENAME_BASE]
+output_file:             @[OUTPUT_FILE]
+ichunk2d:                -1
+jchunk2d:                -1
+ichunk3d:                -1
+jchunk3d:                -1
+kchunk3d:                -1
+ideflate:                @[IDEFLATE]
+nbits:                   @[NBITS]
+write_nemsioflip:        .false.
+write_fsyncflag:         .false.
```

Comment:
please remove write_nemsioflip: and write_fsyncflag:

Reply:
Thanks, @junwang-noaa! These changes have been made accordingly.